core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -248,6 +289,14 @@ public void getStatus() throws Exception {
     return conf;
   }
 
+  private void mockFileSystemContextAndMasterClient() {
+    mMockFileSystemContext = Mockito.mock(FileSystemContext.class);
+    Whitebox.setInternalState(FileSystemContext.class, "INSTANCE", mMockFileSystemContext);
```

Comment:
The right way to do this would be to inject the file system context to directly-created `AbstractFileSystem` instances, instead of creating them with `org.apache.hadoop.fs.FileSystem.get`. If we injected the context we might not need to inject the `FileSystem` any more (it can be created from the context).